book/modules/using_modules.md

```diff
@@ -0,0 +1,227 @@
+# Using Modules
+
+[[toc]]
+
+## Overview
+
+End-users can add new functionality to Nushell by using ("importing") modules written by others.
+
+To import a module and its definitions, we call the [`use`](/commands/docs/use.md) command:
+
+```nu
+use <path/to/module> <members...>
+```
+
+For example:
+
+```nu
+use std/log
+log info "Hello, Modules"
+```
+
+::: tip
+The example above uses the [Standard Library](../standard_library.md), a collection of modules built-in to Nushell. Because it is readily available to all Nushell users, we'll also use it for several of the examples below.
+:::
+
+## Installing Modules
+
+Installing a module is simply a matter of placing its files in a directory. This might be done via `git clone` (or other version control system), a package manager such as `nupm`, or manually. The module's documentation should provide recommendations.
+
+## Importing Modules
+
+Anything after the [`use`](/commands/docs/use.md) keyword forms an **import pattern** which controls how the definitions are imported.
+
+Notice above that `use` has two arguments:
+
+- A path to the module
+- (Optional) The definitions to import
```

Comment:
There's a bit of "technically ..." caveat here ;-)

Technically ... when you `use std/formats *`, you are importing the `std/formats` *module* using a direct path to it.

When you `use std formats *` (which I'm trying to dissuade people from), only *then* are you using a submodule.

So really, the *submodule* goes in the "definitions" argument.

I know kubouch has some ideas on how this might change going forward, but not sure when or if they'll get implemented.